cg/meta/workflow/nf_analysis.py

```diff
@@ -737,3 +739,34 @@ def get_cases_to_store(self) -> list[Case]:
             if self.trailblazer_api.is_latest_analysis_completed(case_id=case.internal_id)
             or self.trailblazer_api.is_latest_analysis_qc(case_id=case.internal_id)
         ]
+
+    def clean_workflow_run_dir(self, yes: bool, case_id: str, dry_run: bool = False):
```

Comment:
```suggestion
    def clean_workflow_run_dir(self, ???: bool, case_id: str, dry_run: bool = False):
```
I am not sure where this `yes` originates from but could this parameter be more descriptive?

Reply:
This is where it originates from:

`OPTION_YES = click.option("-y", "--yes", is_flag=True, help="Skip confirmation")`

Reply:
I think `skip_confirmation` would be better

Reply:
[Issue created](https://github.com/Clinical-Genomics/cg/issues/3184)